app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from pathlib import Path
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+
+    if command == "mv":
+        created_dir = ""
+        new_file_path = list(Path(new_file).parts)
+        new_file_name = old_file
+
+        if "." in new_file_path[-1].strip():
+            new_file_name = new_file_path[-1]
+            new_file_path.pop()
+
+        if len(new_file_path) > 1:
+            for index in range(len(new_file_path)):
+                created_dir = os.path.join(created_dir, new_file_path[index])
+                if not os.path.isdir(created_dir):
+                    os.mkdir(created_dir)
```

Comment:
You can just retrieve dir path with `os.path.dirname` or `os.path.split` and use `os.makedirs` to create directory at once without using a loop